azerothcore/azerothcore-wotlk
# DB/Characters: Remove unused tables "uptime" and "cheaters"

##### CHANGES PROPOSED:

-  Remove table "uptime" from the characters db
-  Remove table "cheaters" from the characters db


###### ISSUES ADDRESSED:


It looks like these tables aren't used at all (always empty). And look at the core:

* uptime: https://github.com/azerothcore/azerothcore-wotlk/search?q=uptime&unscoped_q=uptime
* cheaters: https://github.com/azerothcore/azerothcore-wotlk/search?q=cheaters&unscoped_q=cheaters


##### TESTS PERFORMED:


I verified the code and my DBs but nothing indicate these DBs are used.


##### HOW TO TEST THE CHANGES:


- Verify that I'm not wrong :P
- Verify that these tables are empty on your own servers



##### Target branch(es):

Master
